extensions/blocks/podcast-player/style.scss

```
@@ -1,11 +1,134 @@
 /**
- * Frontend (site) styles for Podcast Episodes
+ * Podcast Player block shared styles (editor & front-end).
```

Really minor. I've seen that we refer to the editor side as `editor-canvas`. Feel free to ignore this commit suggestion.

```suggestion
 * Podcast Player block shared styles (editor-canvas & front-end).
```

---

I couldn't find `editor-canvas` in Jetpack - it's mostly referred to as `block editor`. Could you give an example where it's referenced as the `editor-canvas`, please?